### DISCUSSION TITLE
Support multiple Docker image architectures

### DISCUSSION THREAD

**Discussion by dtcooper** [peers/peerjs#1258] [Ideas & Feature requests]
Support for at least arm64 Docker container would be fantastic! This could be done by modifying the GitHub action. If there's interest and any support for this, I could submit a PR.

**Comment by jonasgloning** [comment]
Yes, that would indeed be great. PR very welcome!

We are using https://github.com/esatterwhite/semantic-release-docker to build the Docker image. Since a few days it finally supports multi-platform builds! 🥳 

I made some quick changes [here](https://github.com/peers/peerjs-server/pull/451). Sadly the run failed: https://github.com/peers/peerjs-server/actions/runs/8513000291/job/23315788634

You can probably take it from there. Thank you for taking this on!
